AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/DataStoreConfiguration.swift

```diff
@@ -0,0 +1,112 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+import Foundation
+
+/// Error Handler function typealias
+public typealias DataStoreErrorHandler = (AmplifyError) -> Void
+
+/// Holds a reference to both the local `Model` and the remote one during a conflict
+/// resolution. Implementations of the `DataStoreConflictHandler` use this to decide
+/// what the outcome of a conflict should be.
+public struct DataStoreConclictData {
```

Comment:
can you update this to ConflictData?

Reply:
I will in a separate commit, the point was to make a little changes between #369 and this PR.